src/protocol.h

```diff
@@ -479,7 +479,7 @@ class CAddress : public CService
         SerReadWriteMany(os, ser_action, ReadWriteAsHelper<CService>(obj));
     }
 
-    //! Always included in serialization, except in the network format on INIT_PROTO_VERSION.
+    //! Always included in serialization.
```

Comment:
that's follow-up changes for https://github.com/bitcoin/bitcoin/pull/20541 which is not backported yet.